config_defaults_inc.php

```diff
@@ -2579,6 +2579,71 @@
 	'view_state',
 );
 
+/**
+ * An array of access levels to identify who should be presented a simplified report form.
+ *
+ * The following Statuses are standard available:
+ *   - VIEWER
+ *   - REPORTER
+ *   - UPDATER
+ *   - DEVELOPER
+ *   - MANAGER
+ *   - ADMINISTRATOR
+ *
+ * @global array $g_simple_report_level
+ */
+$g_simple_report_level = array( REPORTER, UPDATER, );
+
+/**
+ * An array of optional fields to show on the simplified bug report page.
+ *
+ * The following optional fields are allowed:
```

Comment:
No need to provide redundant information.
It's better to use `@see $g_bug_report_page_fields` PHPDoc comment